ui/mirage/factories/dynamic-host-volume.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { Factory } from 'ember-cli-mirage';
+import faker from 'nomad-ui/mirage/faker';
+import { pickOne } from '../utils';
+
+export default Factory.extend({
+  id: () => `${faker.random.uuid()}`,
+  name() {
+    return faker.hacker.noun();
+  },
+
+  pluginID() {
+    return faker.hacker.noun();
+  },
```

Comment:
Not necessary to fix in this PR, but I noticed that the `csi-plugin.js` should probably use this same faker for ID. You're not likely to have UUIDs there.